playbooks/roles/ims_structured_call_interface/tasks/main.yml

```diff
@@ -96,25 +96,26 @@
           cmd: 's {{ DFS_IMS_SSID }}SCI'
           verbose: true
 
-      - wait_for:
+      - name: Wait 10s for previous command to complete
+        wait_for:
           timeout: 10
 
-      - zos_job_output:
-          job_name: '{{ DFS_IMS_SSID }}SCI'
-        register: output
+      # - zos_job_output:
```

Comment:
This code seems to have been updated on PR #18 too. Might cause a conflict when merging both branches to main.